### PR TITLE
Allows ghosts to check silicon laws using examine()

### PR DESCRIPTION
**Pull request by jknpj**
ahelphelper
Doesn't work with astral journey/scrying bastards, tho. Can't do much against the phantom mask right now but it shouldn't be an issue as it gets logged anyways.

:cl:
 * rscadd: You can now check silicon laws as ghost using the examine verb or simply shift+click

### CODE REVIEW

**Comment by Kammerjunk** [comment]
Sure why not
Phantom mask is retarded anyway

File: code/modules/mob/living/silicon/examine.dm [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/mob/living/silicon/examine(mob/user) //Diplay's a silicon's Laws to ghosts
+	if(laws && isobserver(user))
+		var/mob/dead/observer/fag = user
```
Comment (N3X15) [inline code/modules/mob/living/silicon/examine.dm:3]: against gitblub tos and community rules you're under arrest mr

**Comment by SonixApache** [comment]
this is a really cool PR

File: code/modules/mob/living/silicon/examine.dm (hunk 1 above)
Comment (SysadminJeroen) [reply code/modules/mob/living/silicon/examine.dm:3]: GITHUB FEELINGS POLICE PLEASE ARREST THIS MAN

**Comment by Intigracy** [comment]
Wouldn't it be possible to make the phantom mask count them as scrying?

**Pull request merged**